src/test/java/Homework_17.java

```diff
@@ -0,0 +1,73 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.time.Duration;
+
+public class Homework_17 {
+
+//public class Homework17 extends BaseTest {
+
+    @Test
+    public void addSongToPlaylist() {
```

Comment:
If possible, avoid overly long methods. A long method is 20‒25 lines. If possible, you should split the method into several shorter ones or even create a new class for this.